mevislab.github.io/content/tutorials/visualization/visualizationexample8.md

```diff
@@ -0,0 +1,147 @@
+---
+title: "Example 8: Vessel Segmentation using SoVascularSystem"
+date: 2023-12-08
+status: "OK"
+draft: false
+weight: 592
+tags: ["Advanced", "Tutorial", "Visualization", "3D", "Vessel Segmentation"]
+menu: 
+  main:
+    identifier: "visualization_example8"
+    title: "Vessel Segmentation using SoVascularSystem."
+    weight: 592
+    parent: "visualization"
+---
+# Example 8: Vessel Segmentation using SoVascularSystem {#TutorialVisualizationExample8}
+
+## Introduction
+In this tutorial, we'll walk you through the steps to craft a visually striking portrayal of vascular structures by harnessing the capabilities of the `DtfSkeletonization` module. By combining the vessel centerline extraction power of `DtfSkeletonization` with the immersive 3D visualization provided by the `SoVascularSystem` module, you'll effortlessly explore and interact with the intricacies of the resulting vascular system. This integration enhances your ability to comprehensively understand and analyze complex vascular structures.
+
+## Steps to do
+### Develop your network
+
+Load your tree mask ML image using the `LocalImage` module. Then connect the output to the `DtfSkeletonization` module as seen below. Adjust parameters if needed and press the *Open* button to obtain the skeleton and a graph with distinct node and edge IDs.
+
+![LocalImage](/images/tutorials/visualization/V8_1.png "LocalImage")
+
+{{<alert class="info" caption="Extra Infos">}}
+
+Please be aware that a sample tree mask file is included as an attachment for this example
+
+{{</alert>}}
+
+Next, access the `DtfSkeletonization` module's panel and activate *Update skeleton graph* and *Compile graph voxels* for enhanced functionality.
```

Comment:
Explain enhanced functionality, see tooltips for explanations or help of the module. What do this fields do?